core/trino-main/src/test/java/io/trino/operator/output/TestPositionsAppender.java

```diff
@@ -398,6 +404,24 @@ private void testAppend(TestType type, List<BlockView> inputs)
         assertEquals(secondBlock.getPositionCount(), 0);
     }
 
+    private void testAppendSingle(TestType type, List<BlockView> inputs)
+    {
+        PositionsAppender positionsAppender = POSITIONS_APPENDER_FACTORY.create(type.getType(), 10, DEFAULT_MAX_PAGE_SIZE_IN_BYTES);
```

Comment:
could you add a condition to check that Rle block after single-row append does no longer produce rle?

Reply:
done in `testRleAppendedWithSinglePositionDoesNotProduceRle`